jscomp/syntax/native_ast_derive_abstract.ml

```diff
@@ -176,7 +186,11 @@ let code_sig_transform sigi = match sigi with
       } as _makerVb) :: []))
     } ->
     Sig.value (Val.mk ~loc:pstr_loc name typ)
-  | _ -> Sig.type_ []
+  | _ -> 
+#if BS_NATIVE  then
+    Sig.type_ []
+#else assert false    
+#end
 
 let handleTdclsInStr ~light tdcls =
```

Comment:
@bassjacob this module does not work with 4.06, I add some quick hack to make it compile

Reply:
maybe you meant @bsansouci ?

Reply:
Oh that’s weird. Does it compile when you set the BS_NATIVE?